README.md

```diff
@@ -5,52 +5,65 @@
 [![codecov](https://codecov.io/gh/iterative/mlem/branch/main/graph/badge.svg?token=WHU4OAB6O2)](https://codecov.io/gh/iterative/mlem)
 [![PyPi](https://img.shields.io/pypi/v/mlem.svg?label=pip&logo=PyPI&logoColor=white)](https://pypi.org/project/mlem)
 
-MLEM helps you with model deployment. It saves ML models in a standard format that can be used in a variety of downstream deployment scenarios such as real-time serving through a REST API or batch processing. MLEM format is a human-readable text that helps you use GitOps with Git as the single source of truth.
+MLEM helps you package and deploy machine learning models.
+It saves model metadata in a standard, human-readable format that can be used in a variety of deployment scenarios, such as real-time serving through a REST API or a batch processing task.
+MLEM lets you keep Git as the single source of truth for code and models.
 
-- **Run your model anywhere you want:** package it as a Python package, a Docker Image or deploy it to Heroku (SageMaker, Kubernetes and more platforms are coming). Switch between formats and deployment platforms with a single command thanks to unified abstraction.
-- **Simple text file to save model metadata:** automatically package Python env requirements and input data specifications into a ready-to-deploy format. Use the same human-readable format for any ML framework.
-- **Stick to your training workflow:** MLEM doesn't ask you to rewrite your training code. To start using packaging or deployment machinery, add just two lines to your python script: one to import the library and one to save the model.
-- **Developer-first experience:** use CLI when you feel like DevOps and API when you feel like a developer.
+- **Run your ML models anywhere:**
+  Wrap models as a Python package or Docker Image, or deploy them to Heroku (SageMaker, Kubernetes, and more platforms coming soon).
+  Switch between platforms transparently, with a single command.
 
-## Why MLEM?
+- **Simple text file to save model metadata:**
+  Automatically include Python requirements and input data needs into a deployment-ready format.
+  Use the same format on any ML framework.
```

Comment:
Is the format YAML? Where can we see the schema?

Reply:
`.mlem` yaml schema is dependent on the type of the object. For model, data, deployment, etc, it will be a different schema. IIRC, this is the reason why we cannot easily create the schema as DVC can. We can take a look later, but this is not the priority rn.

Reply:
Created the issue to address this later https://github.com/iterative/mlem/issues/306

Reply:
OK np. Putting a pseudo schema spec somewhere in mlem.ai.doc would be nice though ⏳